pennylane/ops/mid_measure/pauli_measure.py

```diff
@@ -31,8 +31,6 @@
 class PauliMeasure(Operator):
     """A Pauli product measurement."""
 
-    resource_keys = {"pauli_word"}
```

Comment:
I'm removing this because `PauliMeasure` is not going to be integrated into the decomposition graph. As far as the graph is concerned, it does not require any additional knowledge of the `PauliMeasure` because it will never try to find a decomposition for it.